nexong/api/Lesson/lessonSerializer.py

```diff
@@ -30,17 +30,18 @@ def validate(self, attrs):
         validation_error = {}
 
         max_attendees = attrs.get("capacity")
-        if max_attendees < 1:
-            validation_error["capacity"] = "capacity must be higher than 0."
-        attendees = attrs.get("students")
-        if attendees:
-            num_attendees = len(attendees)
-        else:
-            num_attendees = 0
-        if max_attendees < num_attendees:
-            validation_error[
-                "capacity"
-            ] = "capacity must be higher or equal to the number of attendees selected."
+        if max_attendees is not None:
+            if max_attendees < 1:
+                validation_error["capacity"] = "capacity must be higher than 0."
+            attendees = attrs.get("students")
+            if attendees:
+                num_attendees = len(attendees)
+            else:
+                num_attendees = 0
+            if max_attendees < num_attendees:
+                validation_error[
+                    "capacity"
+                ] = "capacity must be higher or equal to the number of attendees selected."
```

Comment:
This does not exactly work. I patched the capacity field with 1, but the lesson already contains 2 students and an error is not raised. The reason is that if capacity is patched, only capacity is sent to backend, so, `attrs.get('students')` is null.